main.js

```diff
@@ -4359,19 +4393,54 @@ function init(compactGroupId) {
     try {
         pluginHandler.addPlugins(config.plugins, __dirname);           // ... plugins from iobroker.json
     } catch (e) {
-        logger.error('Cannot load plugins ' + JSON.stringify(config.plugins) + ': ' + e);
-        console.error('Cannot load plugins ' + JSON.stringify(config.plugins) + ': ' + e);
+        logger.error(`${hostLogPrefix} Cannot load plugins ${JSON.stringify(config.plugins)}: ${e}`);
+        console.error(`Cannot load plugins ${JSON.stringify(config.plugins)}: ${e}`);
     }
 
     createObjects(() => {
         objects.subscribe('system.adapter.*');
 
         // create states object
-        createStates(() => {
+        createStates(async () => {
             // Subscribe for connection state of all instances
             // Disabled in 1.5.x
             // states.subscribe('*.info.connection');
 
+            let notificationsConfig;
+            try {
+                const obj = await objects.getObjectAsync(`system.host.${hostname}`);
+
+                if (obj && obj.notifications) {
```

Comment:
nicht besser direkt aus der io-package holen? :-) ist das host object immer up to data?

Reply:
ah okay, missed that we anyway need to read io-pack so the call is pretty redundant